### ISSUE TITLE
Inline attribute will always include the script inline regardless of the value.

### ISSUE THREAD

**Issue by xupengqi** [raptorjs/optimizer#43]
Will include inline script:
```
    { "path": "./lib/script.js", "slot": "body", "inline": false }
    { "path": "./lib/script.js", "slot": "body", "inline": true }
```
Will include script as a resource:
```
    { "path": "./lib/script.js", "slot": "body" }
```

**Comment by philidem** [comment]
Good catch. The `inline` attribute should support these values:
- `"end"` (creates a `<SCRIPT>` after all of the non-inlined `<SCRIPT src="">` blocks)
- `"in-place"` (creates a `<SCRIPT>` block in-between other `<SCRIPT src="">` blocks)
- `"beginning"` (creates a `<SCRIPT>` before all of the non-inlined `<SCRIPT src="">` blocks)
- `true` / `"true"` (same as `"end"`)
- `false` / `"false"` (should not end line the script -- _this is broken_)

As you discovered, the `false` / `"false"` case was not being handled correctly and was being treated as `"end"`. I'll fix it so that `false` / `"false"` does not inline the resource.

**Comment by patrick-steele-idem** [comment]
Thanks, @philidem, for looking into the problem and working on a fix.

**Comment by philidem** [comment]
Published **optimizer 1.7.1** which fixes this issue
